src/script/Container/runContainer.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# runContainer.sh: script to pull specified CRAB TW image from defined repo and run it
+
+helpFunction(){
+  echo -e "\nUsage example: ./runContainer.sh -v v3.201118 -s TaskWorker"
+  echo -e "\t-v TW/Publisher version"
+  echo -e "\t-s which service should be started: Publisher, Publisher_scheddy, Publisher_asoless, Publisher_rucio or TaskWorker"
+  echo -e "\t-r docker hub repo, if not provided, default points to 'cmssw'"
+  exit 1
+  }
+
+while getopts ":v:s:r:h" opt
+do
+    case "$opt" in
+      h|--help) helpFunction ;;
```

Comment:
I do not think that `|--help` does anything here. If on indicate `--help` or anything of the stort `--string` the script always reports `Unimplemented option: --`

Reply:
you are right, thanks.